xenserver/PXELinux.erb

```diff
@@ -0,0 +1,12 @@
+<%#
+kind: PXELinux
+name: XenServer default PXELinux
+oses:
+- XenServer
+%>
+
+default xenserver
+label xenserver
+kernel mboot.c32
+append <%= @xen %> dom0_max_vcpus=16 dom0_mem=752M com1=115200,8n1 console=com1,vga --- <%= @kernel -%> xencons=hvc console=hvc0 console=tty0 answerfile=<%= foreman_url('provision')%> install --- <%= @initrd %>
```

Comment:
Are ```dom0_max_vcpus``` and ```dom0_mem``` settings that should typically be modified? If so, should they be parameters?